src/pydata_sphinx_theme/assets/styles/sections/_header-article.scss

```diff
@@ -4,13 +4,6 @@
   // The items define the height so that it disappears if there are no items
   .header-article-item {
     min-height: var(--pst-header-article-height);
-    height: var(--pst-header-article-height);
```

Comment:
Removing the height rule should prevent the issue in #1451, in which the breadcrumbs overlap the article title